packages/buffetjs-styles/src/assets/styles/sizes.js

```diff
@@ -13,6 +13,10 @@ const sizes = {
       small: '15px',
       large: '15px',
     },
+    minWidth: {
+      small: '140px',
+      large: '140px',
```

Comment:
Is `minWidth.large` needed? Seems never used

Reply:
Just for consistency

Reply:
We also can just set `minWidth: 140px` but alright!